NEWS

```diff
@@ -84,6 +84,10 @@ SOAP:
 Sockets:
   . Removed the deprecated inet_ntoa call support. (David Carlier)
 
+Sodium:
+  . Add support for AEGIS-128L and AEGIS-256 (jedisct1)
+  . Enable AES-GCM on aarch64 with the ARM crypto extensions (jedisct1)
+
```

Comment:
Can you update the UPGRADING document too, and then we are good to merge :)

Reply:
Should that go to the PHP 8.3 branch first?

Reply:
Nope, just master as, sadly, the 8.3 ship has sailed. So this will only be available in 8.4 :/